src/Options.tsx

```diff
@@ -12,65 +11,10 @@ export type NavigationBarConfig = {
 
 export type StatusBarConfig = {
     lightStatusBar?: boolean,
-    /**
-     * Android only option. Have no effect at iOS.
-     */
-    statusBarColor?: string,
+    statusBarColor?: string, // Android only option. Have no effect at iOS. 
 };
 
 /**
  * Option: Data points
  */
 export type DataPoints = { [key: string]: string };
-
-// ----------------------- Deprecated -------------------------------
-
-export type ConfigOptionName = "isS2S" | "userId" | "sessionUid" | "dataPoints";
-export type StylingOptionName = "title" | "language" | "navigationBar" | "statusBar";
-export type NavigationBarOptionName = "backgroundColor" | "buttonsColor" | "titleColor" | "hasShadow";
-export type StatusBarOptionName = "lightStatusBar";
-
-/**
- * @deprecated Please, use `setSessionID` and `setUserID` functions instead
- */
-export type ConfigOptionTypes = {
```

Comment:
Let's make similar to the methods - remove all from the inside but leave the type declaration with @unsupported annotation and the description of what to use instead

Reply:
done